components/about/Team.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="about-team">
+    <AboutSvgWavyBump top />
+    <div class="about-team-content">
+      <v-container>
+        <v-row>
+          <h2>Meet the Team</h2>
+        </v-row>
+        <v-row>
+          <v-col cols="12" sm="12" md="6" lg="6" v-for="lead in leads" :key="lead.name">
+            <AboutTeamMember
+              :github-handle="lead.githubHandle"
+              :twitter-handle="lead.twitterHandle"
+              :name="lead.name"
+              :role="lead.role"
+              :custom-pic="lead.customPic"
+            />
+          </v-col>
+          <v-col sm="12" md="6" lg="6">
+            <div class="about-team-more-contributors">
+              <div>
+                <h3>Plus many more members who have lent us a helping hand!</h3>
+                <br />
+                <h3>Find them on our <a href="https://github.com/ourjapanlife/findadoc-frontend">front end</a> and <a href="https://github.com/ourjapanlife/findadoc-localization">localization</a> Github contributors pages</h3>
+              </div>
+            </div>
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col sm="12">
+            <div class="about-team-join">
+              <h3>Does Find a Doc’s mission spark your interests?</h3>
+              <h3>Join us over on GitHub to see how you can contribute!</h3>
+              <h3><a href="https://github.com/ourjapanlife">✋ I want to join!</a></h3>
+            </div>
+          </v-col>
+        </v-row>
+      </v-container>
+    </div>
+    <AboutSvgWavyBump />
+  </div>
+</template>
+<script>
+import { leads } from "../../data/contributors.json";
+
+export default {
+  data() {
+    return {
+      leads: leads
```

Comment:
should avoid using identical variable names. 
maybe call the left assignment, `teamMembers` to match the component context.

Reply:
Sure? Since the context is different there's no collision, but I can change.